### PR TITLE
Small minor cleanups

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

**Comment by shabanovd** [comment]
@adamretter Is it same PR or new one? All comments washout 3 times now.

**Comment by adamretter** [comment]
@shabanovd Same except changes to `[refactor] minor cleanup` as you suggested. Not sure what to do about the `[refactor] Do not config collections twice` one yet, can you comment on the thread on the mailing-list please, and then you comments won't be lost when I rebase.

**Comment by adamretter** [comment]
I moved the double-collection-config fix to a separate branch https://github.com/eXist-db/exist/pull/734 so as not to delay this one further.

**Comment by shabanovd** [comment]
@adamretter this one is also ok, comment was on minor cleanup (move synchronized to method signature), so if everyone happy it can go.

**Comment by adamretter** [comment]
@shabanovd @ljo If happy, please merge...

**Comment by ljo** [comment]
Yes, I am happy.

**Comment by adamretter** [comment]
@ljo please can you merge then

File: src/org/exist/storage/lock/DeadlockDetection.java [hunk 1]
```diff
@@ -21,6 +21,10 @@
  */
 package org.exist.storage.lock;
 
+import org.apache.log4j.LogManager;
```
Comment (dizzzz) [inline src/org/exist/storage/lock/DeadlockDetection.java]: how can this work? this is log4j 1.2.x right?

Comment (dizzzz) [inline src/org/exist/storage/lock/DeadlockDetection.java]: answering myself...  because of the log4j compatibility jar in lib/user ....

Comment (adamretter) [inline src/org/exist/storage/lock/DeadlockDetection.java]: Ah crap! There is an excellent example of why we shouldn't of added it!

**Pull request merged**